torchtext/experimental/datasets/raw/text_classification.py

```diff
@@ -55,15 +55,9 @@ def __iter__(self):
             self.setup_iter()
 
         for i, item in enumerate(self._iterator):
-            if i == self.start:
```

Comment:
Are those changes to resolve the errors for sync? Or I think we should do it in a separate PR.

Reply:
The lint complained that ```item``` wasn't used even though it is defined. The follow-up code relies on the idea that item will be populated by the loop iteration, which I'm not even sure will happen if self._iterator where to already be depleted, which would then raise an error on line 63.